codalab/worker/main.py

```diff
@@ -209,9 +209,13 @@ def main():
 
     # Configure logging
     logging.basicConfig(
-        format='%(asctime)s %(message)s', level=(logging.DEBUG if args.verbose else logging.INFO)
+        format='%(asctime)s %(message)s %(pathname)s %(lineno)d',
+        level=(logging.DEBUG if args.verbose else logging.INFO),
     )
 
+    if not args.verbose:
+        logging.getLogger('urllib3').setLevel(logging.INFO)
```

Comment:
Can you make a separate PR that just does this change, and silences the urllib3 logs even when the worker is set to verbose?

Reply:
https://github.com/codalab/codalab-worksheets/pull/3280